templates/new-feature-email.html

```diff
@@ -15,7 +15,7 @@
 
 <p><b><a href="https://www.chromestatus.com/feature/{{id}}"
          >{{feature.name}}</a></b>
-  (added {{feature.created|date}})</p>
```

Comment:
Should this also be removed in the code that injects the date string as a field into the object?

Reply:
The python code does not specifically add the date field to the template data dictionary.  It used to do that but I already removed it in a previous CL.  The current code accesses a field of the Feature object, and that Feature object is needed for other parts of the template.